examples/mobile/UIComponents/index.html

```diff
@@ -20,68 +20,68 @@ <h1>
 			</h1>
 		</header>
 		<div class="ui-content">
-			<ul class="ui-listview ui-colored-list">
-				<li>
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-anchor">
```

Comment:
I just have a question.
If it use with divider, is it right  as     li class="ui-li-divider ui-li-anchor" ?

Reply:
Yes, It is right. But I don't think the 'ui-li-divider' is used by default.

Reply:
I think this is sth what we need to clarify with UX if divider is used by default or not

Reply:
In the UX , 
List basically has a divider.
But in case of the list used in Dialog, it basically doesn't have a divider.
User have a choice by "ui-li-divider" and now there is no problem for it.